### DISCUSSION TITLE
APPCRASH Brackets.exe c000041d

### DISCUSSION THREAD

**Discussion by serioznochtoliumno45372** [brackets-cont/brackets#307] [Q&A]
Description
Wrong application path: C:\Program Files (x86)\Brackets\Brackets.exe

Problem signature
Name of the problematic event: APPCRASH
Application Name: Brackets.exe
Application Version
: 1.14.2.0 Application Timestamp: 5e82f6b5
The name of the module with the error: Brackets.exe
Version of the module with an error:	1.14.2.0
Module timestamp with error: 5e82f6b5
Exception Code: c0000005
Exception Offset: 00018806
OS Version:	10.0.19045.2.0.0.256.48
Language code: 1049
Additional information 1: 9744
Additional information 2: 97448724f6c837c524967a609ca1975c
Additional information 3: c9 CE
Additional information 4: c9ce376a30b70db878dd459d08146582

More information about the problem
Container ID: 58f120a02fb67579428ff7aaeb509dac (1337559927816756652)

The path of the application location is correct, I tried reinstalling, restarting, downloading all the latest updates, but the application does not start and this is how it happens: "I click start the brackets application in windows 10, the brackets process appears in the background processes in the task manager, but after a short time this process disappears and the application does not start"

**Comment by charlypa** [comment]
please download latest version from
https://brackets.io/

**Reply by serioznochtoliumno45372** [reply]
I have the latest version of Brackets installed, I have been waiting for a while when a new version of the application was released, but the error still remains. I've been trying to solve it for more than half a year, but I haven't asked any questions on the forum before. I decided to ask a question after learning that all my friends use the application on older, and newer, and even similar builds absolutely without problems.
